src/Components/RequestDataScreen/IntervalsInfo.tsx

```diff
@@ -160,11 +160,11 @@ export class IntervalsInfo extends React.Component<Props, State> {
 
     renderDescription() {
         return <Box>
-            Query an Account's electricity or gas usage intervals, down to a fifteen-minute granularity.
+            Query an Account's electricity or gas usage intervals, up to fifteen minute granularity.
```

Comment:
nit: "up to" sounds kinda awkward given that 15 min is the _smallest_ granularity. But not a big deal tbh